phpcs.xml.dist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<ruleset name="Custom ruleset for Liveblog">
+
+	<rule ref="WordPress-Extra"/>
```

Comment:
Also add `WordPress-Docs`? It's not fully complete, but it's a good start for what it does cover. Could leave for a separate PR.

Reply:
Yep, let's do a separate issue outside  of this PR :+1: